.github/workflows/pr.yml

```diff
@@ -16,7 +16,7 @@ jobs:
 
     strategy:
       matrix:
-        os: [ubuntu-latest, macos-latest, macos-13-xlarge]
+        os: [ubuntu-latest, macos-latest, macos-13-xlarge, windows-latest]
```

Comment:
May be we'd need to use platform agnostic line endings in our tests? 

```
#[cfg(windows)]
const LINE_ENDING: &'static str = "\r\n";
#[cfg(not(windows))]
const LINE_ENDING: &'static str = "\n";
```


https://stackoverflow.com/a/47541878/3452318

Reply:
This is not what we need here, since its the fixtures throwing us off.